spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] After setting up your project and running the tests, almost all the tests fail, I'll suggest you take a look at this as your unit tests need to pass to ensure the efficiency of your code. :+1: :100: 

-  Screenshot reference:
![Screenshot from 2023-11-30 21-24-33](https://github.com/demesameneshoa/blog-app/assets/107632780/ab193e0e-6ecf-4d0c-9e4c-b146eff8ffab)
![Screenshot from 2023-11-30 21-22-43](https://github.com/demesameneshoa/blog-app/assets/107632780/bf4c24b2-ec48-4c98-b7c2-6111c0df258f)